railties/test/generators/scaffold_generator_test.rb

```diff
@@ -179,6 +179,10 @@ def test_system_tests_without_attributes
     assert_file "test/system/product_lines_test.rb" do |content|
       assert_match(/class ProductLinesTest < ApplicationSystemTestCase/, content)
       assert_match(/test "visiting the index"/, content)
+      assert_match(/assert_selector "h1", text: "Product line"/, content)
+      assert_match(/click_on "Show this product line"/, content)
+      assert_match(/click_on "Edit this product line"/, content)
+      assert_match(/click_on "Destroy this product line"/, content)
```

Comment:
Ideally, these content assertions could be replaced by a separate test that _executes_ the generated System Tests within a dummy application context.

Is there existing test architecture that makes that happen?